onebusaway-android/src/main/java/org/onebusaway/android/io/elements/ObaRegionElement.java

```diff
@@ -285,6 +293,17 @@ public boolean getSupportsSiriRealtimeApis() {
     }
 
     @Override
+    public boolean getSupportsEmbeddedSocial() {
```

Comment:
I'd prefer not to mix a device-specific setting (whether or not the account restrictions allow social) with the IO Region model object.  Could you please remove the `embeddedSocialEnabled` here?  I think a utility method to pull the info directly would probably be best, but I'll comment on that elsewhere where the data is accessed.